README.md

```diff
@@ -41,7 +42,7 @@ available at [http://localhost:3000/](http://localhost:3000/).
 
 #### Unit Test
 
-The unit tests are powered by [chai](http://chaijs.com/) and [mocha](http://mochajs.org/).
+The unit tests are powered by [Jest](https://facebook.github.io/jest/), [Enzyme](http://airbnb.io/enzyme/) and [Sinon.JS](http://sinonjs.org/)
```

Comment:
Yes. But the jest has lots of advantage in React unit testing. Jest is recommended by Facebook to do unit test for React. And react has code coverage integrated, mocha needs a 3rd package to implement code coverage. The snapshot feature in Jest is very useful for React unit test. So I think we need to keep Jest to run unit test.